policy/pipeline/required_tasks.rego

```diff
@@ -73,17 +73,22 @@ deny contains result if {
 #   is not currently included in the Pipeline definition.
 # custom:
 #   short_name: missing_future_required_task
-#   failure_msg: '%s is missing and will be required in the future'
+#   failure_msg: '%s is missing and will be required on %s'
 warn contains result if {
 	count(tkn.tasks(input)) > 0
 
 	# Get missing tasks by comparing with the default required task list
-	some required_task in _missing_tasks(latest_required_tasks)
-
-	# If the required_task is also part of the current_required_tasks, do
-	# not proceed with a warning since that's clearly a violation.
-	not required_task in current_required_tasks
-	result := lib.result_helper_with_term(rego.metadata.chain(), [_format_missing(required_task, true)], required_task)
+	# this is empty
+	some required_task in _missing_tasks(tkn.required_tasks(latest_required_tasks))
+
+	# # If the required_task is also part of the current_required_tasks, do
+	# # not proceed with a warning since that's clearly a violation.
```

Comment:
```suggestion
	# If the required_task is also part of the current_required_tasks, do
	# not proceed with a warning since that's clearly a violation.
```